### ISSUE TITLE
lazy import does not package in pyinstaller easilly

### ISSUE THREAD

**Issue by gsemet** [mnmelo/lazy_import#5]
Hello

Easy import works great and does a marveillous job, however I had to fork and disable the VERSION file trick in order to package it properly in pyinstaller, since this guy does not take full package but extract only python files from the dependencies.

Can you update to be pyinstaller friendly? Lazy import is really cool to avoid unneeded module load in CLI tools using pyinstaller.

Thanks!

**Comment by smerkousdavid** [comment]
@gsemet I totally agree with this and would love to see that file removed because it causes issues when trying to package this library with any packaging without either modifying and/or moving the file. @mnmelo Can we get an update on this repo?
